error Error: Cannot find module '/var/folders/2y/.../T/rncli-init-template-jUussB/node_modules/typescript/template.config'

# Question

When trying to init a new project I get the following error message:
```
~/MyDev/test$ npx react-native init myapp --template typescript
                                                          
               ######                ######               
             ###     ####        ####     ###             
            ##          ###    ###          ##            
            ##             ####             ##            
            ##             ####             ##            
            ##           ##    ##           ##            
            ##         ###      ###         ##            
             ##  ########################  ##             
          ######    ###            ###    ######          
      ###     ##    ##              ##    ##     ###      
   ###         ## ###      ####      ### ##         ###   
  ##           ####      ########      ####           ##  
 ##             ###     ##########     ###             ## 
  ##           ####      ########      ####           ##  
   ###         ## ###      ####      ### ##         ###   
      ###     ##    ##              ##    ##     ###      
          ######    ###            ###    ######          
             ##  ########################  ##             
            ##         ###      ###         ##            
            ##           ##    ##           ##            
            ##             ####             ##            
            ##             ####             ##            
            ##          ###    ###          ##            
             ###     ####        ####     ###             
               ######                ######               
                                                          

                  Welcome to React Native!                
                 Learn Once Write Anywhere                

✔ Downloading template
✖ Copying template
error Error: Cannot find module '/var/folders/2y/kvrr5nws6xz4nx9vnwjxj17w0000gn/T/rncli-init-template-jUussB/node_modules/typescript/template.config'
Require stack:
- /Users/doberkofler/.npm/_npx/27475/lib/node_modules/react-native/node_modules/@react-native-community/cli/build/commands/init/template.js
- /Users/doberkofler/.npm/_npx/27475/lib/node_modules/react-native/node_modules/@react-native-community/cli/build/commands/init/init.js
- /Users/doberkofler/.npm/_npx/27475/lib/node_modules/react-native/node_modules/@react-native-community/cli/build/commands/init/index.js
- /Users/doberkofler/.npm/_npx/27475/lib/node_modules/react-native/node_modules/@react-native-community/cli/build/commands/index.js
- /Users/doberkofler/.npm/_npx/27475/lib/node_modules/react-native/node_modules/@react-native-community/cli/build/cliEntry.js
- /Users/doberkofler/.npm/_npx/27475/lib/node_modules/react-native/node_modules/@react-native-community/cli/build/index.js
- /Users/doberkofler/.npm/_npx/27475/lib/node_modules/react-native/cli.js
```

--- a/README.md
+++ b/README.md
@@ -25,6 +25,14 @@
 
 ## :arrow_forward: Usage
 
+### React Native >= 0.60
+
+```sh
+npx react-native init MyApp --template react-native-template-typescript@next
+```
+
+### React Native <= 0.59
+
 ```sh
 react-native init MyApp --template typescript
 ```

